packages/codegen-ui/lib/types/string-format.ts

```diff
@@ -14,6 +14,21 @@
   limitations under the License.
  */
 
+export enum DATE {
```

Comment:
Can we please not use enums? It adds more work when we have to deserialize. I'd rather we just have a type that represents the available string options.